src/ai/backend/manager/server.py

```diff
@@ -438,6 +448,11 @@ async def event_dispatcher_ctx(root_ctx: RootContext) -> AsyncIterator[None]:
         root_ctx.shared_config.data["redis"],
         db=REDIS_STREAM_DB,
     )
+    try:
+        await root_ctx.event_producer.ping()
+    except (Exception, asyncio.CancelledError) as e:
+        log.error("Failed to initiate event producer (error: {0})", repr(e))
+        raise
```

Comment:
Umm… Rather than inserting `log.error` in every place where an Exception might occur, it seems better to convey exceptions meaningfully and handle them in a common place.

Reply:
Which place should the error handler be? I want the logger logs the exception raised in bootstrap steps